crates/sui-bridge-indexer/src/config.rs

```diff
@@ -24,6 +25,9 @@ pub struct IndexerConfig {
     pub eth_sui_bridge_contract_address: String,
 
     pub metric_port: u16,
+
+    /// A temporary flag to disable the eth indexer to test mainnet before eth contracts are deployed.
+    pub disable_eth: Option<bool>,
```

Comment:
I assume this is because that is the only way to have a default value in the yaml?

Reply:
wdym?